dashboard/src/containers/LoginFormContainer/LoginFormContainer.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { shallow } from "enzyme";
+import { Location } from "history";
+import * as React from "react";
+import { IAuthState } from "reducers/auth";
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import LoginForm from "./LoginFormContainer";
+
+const mockStore = configureMockStore([thunk]);
+
+const makeStore = (
+  authenticated: boolean,
+  authenticating: boolean,
+  authenticationError: string,
+) => {
+  const state: IAuthState = {
+    authenticated,
+    authenticating,
+    authenticationError,
+  };
+  return mockStore({ auth: state });
+};
+
+const emptyLocation: Location = {
+  hash: "",
+  pathname: "",
+  search: "",
+  state: "",
+};
+
+describe("LoginFormContainer props", () => {
+  it("maps authentication redux states to props", () => {
+    const store = makeStore(true, true, "It's a trap");
```

Comment:
why not just pass the IAuthState object directly? not sure why this helper is valuable, it's also not easy to read what the first two parameters relate to

Reply:
also, shouldn't we test different variations?

Reply:
I do not see much value on passing variations since the mapStateToProps just takes the values, there are not defaults or anything.